middleware/varnish/README.md

```diff
@@ -0,0 +1,57 @@
+# MIDDLEWARE Varnish SignalFx detectors
+
+## Prerequisites
+
+This monitor uses the **varnishstat** command. The **signalfx-agent** user must be
+able to use this command. In order to do that, you need to start **varnishncsa**.
+On **Varnish 4**, you need to add in the **/etc/default/varnishncsa** file :
```

Comment:
In my test with varnish 5 & 6 I do not need to modify or start varnishncsa. do you confirm it is only for varnish 4 ?